docs/documentation/server_admin/topics/organizations/managing-members.adoc

```diff
@@ -86,7 +86,6 @@ intention to join the organization.
 On the other hand, if no user is associated with the given email address, the steps
 will involve creating a new account through the realm's self-registration flow. In this case, the user is forced
 to provide the same email address used to send the invitation.
-Make sure to enable the *User registration* setting in the *Login* tab at the *Realm settings* to use this type of invitation.
```

Comment:
This change should be included in 26 regardless of what we do with this PR, right? If so I would open a separate PR to ensure we include this in the final documentation

Reply:
https://github.com/keycloak/keycloak/pull/33411